### ISSUE TITLE
Export-->Animator + selected AnimationClips not exporting mesh

### ISSUE THREAD

**Issue by JackHarris1973** [RazTools/Studio#100]
Hi, I'm trying to export Animator/AnimationClip combinations from an assortment of games, and no matter the game, I get a single .FBX file that's usually around 25k in size, and can't be opened.  

For each of these games, I've specified the Unity version.  I'm not sure what else to do; it's probably some setting that I have wrong, but I can't seem to figure it out.

**Comment by Modder4869** [comment]
dont export that many animation and disable collect animation in settings and follow the issue template

**Comment by JackHarris1973** [comment]
Sorry for not following the issue template.

I'm exporting only a single AnimationClip with my Animator asset.  I disabled Collect Animation, but it's still not working.  The exported FBX doesn't contain a mesh or an armature, only an animation file that is significantly shorter than it should be.

**Comment by Modder4869** [comment]
and does it export normally without animation? what are you using to check the models

**Comment by Modder4869** [comment]
you need to either load all files and follow the guide in readme properly and build cabmap so it load all dependencies when you load your file

**Comment by redsigma** [comment]
like modder4869 said, you need to Load meshes too when you load animationClips. If not then the animationClips will contain invalid bone names.  

Also from my experience, meshes can have weird names and depending on the keywords that you used to load files from the generated cab,  there is a chance that not all required meshes are loaded, and this can result in what you see where exporting a mesh gives you a 15kb file which probably contains only some anchor points. The ideal case is to load everything but you'll need a lot of ram or hdd space.

**Comment by JackHarris1973** [comment]
I'm just trying to work my way through building the AssetMap.  This is new to me; I've never done this before.

**Comment by Roachdaripper** [comment]
you're extracting the wrong animator, there's usually multiple copies of animators for some reason and only one of them actually contains the mesh

**Comment by JackHarris1973** [comment]
> you're extracting the wrong animator, there's usually multiple copies of animators for some reason and only one of them actually contains the mesh

No, there's only one Animator file for this character.  I've tested this version of AssetRipper with other Unity files that I know for sure are working, and I'm getting the same result.  It has to be some setting I don't understand.

**Comment by JackHarris1973** [comment]
[Bashir_test.zip](https://github.com/RazTools/Studio/files/15279948/Bashir_test.zip)

Here's the file I've been using.
